hashicorp/boundary-ui
# Icu 12587 add filtering pagination support for sessions

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12587)

Also closes Jira ticket for route-action migration for sessions route in [Admin UI](https://hashicorp.atlassian.net/browse/ICU-12609)

## Description



## Screenshots (if appropriate):
Initial screen:
<img width="1251" alt="Screenshot 2024-03-21 at 10 37 36 AM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/b8a5dc9d-51f5-4f83-b89c-13998399b4fe">
Searching by session Id:
<img width="1251" alt="Screenshot 2024-03-21 at 10 38 31 AM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/ce41301d-e9e4-4432-b4f9-de4111394808">
Filtering by target:
<img width="1251" alt="Screenshot 2024-03-21 at 10 38 42 AM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/b724afa2-8a35-401f-bdb5-fdc5cebe80f2">


## How to Test

Start a local `boundary dev` instance and start a few sessions. Navigate to the sessions tab in the project view. You should be able to search and filter the sessions along with paginate.





:technologist: [Admin preview](https://boundary-cbw9ykpbq-hashicorp.vercel.app/scopes/s_1sheljmjgm/sessions)

## Checklist:
